package.json

```diff
@@ -39,9 +43,13 @@
     "@angular/http": "~7.2.1",
     "@angular/platform-browser": "~7.2.1",
     "@angular/platform-browser-dynamic": "~7.2.1",
+    "@angular/platform-server": "~7.2.1",
     "@angular/router": "~7.2.1",
+    "@nguniversal/express-engine": "~7.1.1",
+    "@nguniversal/module-map-ngfactory-loader": "~7.1.1",
     "core-js": "^2.6.2",
-    "intersection-observer": "^0.5.1",
+    "express": "^4.15.2",
+    "intersection-observer": "^0.7.0",
```

Comment:
This is needed because they add some fixes that are needed for the SSR fix?

Reply:
This is to avoid the `window is not defined` exception. They add fixes for the SSR fix. 😄